frontend/src/components/login/login-input/LoginInput.tsx

```diff
@@ -16,13 +16,16 @@ import {
   sendUserEvent,
   sendException,
 } from 'services/ga.service'
+import { i18n } from 'locales'
+import { t } from '@lingui/macro'
+import { Trans } from '@lingui/react'
 
-const emailText = 'Sign in with your gov.sg email'
-const otpText = 'One-Time Password'
-const emailButtonText = ['Get OTP', 'Sending OTP...']
-const otpButtonText = ['Sign In', 'Verifying OTP...']
-const emailPlaceholder = 'e.g. postman@agency.gov.sg'
-const otpPlaceholder = 'Enter OTP'
+const emailText = i18n._(t`Sign in with your gov.sg email`)
```

Comment:
Hm, didn't intend to use lingui this way. Our code currently passes the text as params into a render function because we didn't have an interpolation package. I imagine with the interpolation package, we no longer need to pass translated text as params, but translate them in the component.

Reply:
@jeantanzj Yup agree, will make the change accordingly.

Reply:
I have made the changes, let me know if that works.

Note: based on https://github.com/lingui/js-lingui/issues/332#issuecomment-423121901, extractor can only statically analyse the code and cannot extract the actual string in variables.